aspnet/mvc/controllers/testing/sample/TestingControllersSample/tests/TestingControllerSample.Tests/IntegrationTests/SessionControllerIndex.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.PlatformAbstractions;
+using TestingControllersSample;
+using Xunit;
+
+namespace TestingControllerSample.Tests.IntegrationTests
+{
+    public class SessionControllerIndex
```

Comment:
There seems to be a separate file for each method on the controller. This seems kind of unusual. Can you please have  a single file for all the methods in a controller instead? (this is typically how we write tests in MVC or other repos)